### DISCUSSION TITLE
Calculation of source term: Sbs: Bottom scattering

### DISCUSSION THREAD

**Discussion by krober10nd** [NOAA-EMC/WW3#772] [Q&A]
Hello all, I'd like to use source term to described in the WW3 manual in section 2.3.19 regarding bottom scattering. 

This requires the calculation of the term Fb which describes the bottom spectrum variance as a function of wavenumber. I'd like to know if there any existing functions/routines or perhaps just an examples of a file? It's mentioned this can be calculated through bathymetry so I'd be curious if anyone can help me figure out a methodology to calculate this. 

![image](https://user-images.githubusercontent.com/18619644/186675254-da994d68-16fc-4212-930c-e60567d23aff.png)

Thank you for any help you can provide!

**Comment by aliabdolali** [comment]
@krober10nd  I think regtests/ww3_tp1.10 has such a file. 
https://github.com/NOAA-EMC/WW3/blob/develop/regtests/ww3_tp1.10/info

**Comment by krober10nd** [comment]
Thanks @aliabdolali for pointing the input file. Any thoughts on how to calculate this quantity?
